code/modules/food_and_drinks/kitchen_machinery/processor.dm

```diff
@@ -215,7 +232,7 @@
 	for(var/O in src.contents)
 		var/datum/food_processor_process/P = select_recipe(O)
 		if (!P)
-			log_admin("DEBUG: [O] in processor havent suitable recipe. How do you put it in?") //-rastaf0 // DEAR GOD THIS BURNS MY EYES HAVE YOU EVER LOOKED IN AN ENGLISH DICTONARY BEFORE IN YOUR LIFE AAAAAAAAAAAAAAAAAAAAA - Iamgoofball
```

Comment:
Well I am sorry that you feel that way but keeping something like this is silly in live code. There is still the git history so it dosent really mean that its gone. Anyhow history is when we run old versions for fun.